deploy/chart/values.yaml

```diff
@@ -55,3 +61,6 @@ package:
 monitoring:
   enabled: false
   namespace: monitoring
+
+pprof:
+  enabled: false
```

Comment:
Isn't this technically always enabled now (though effectively disabled if no client CA bundle is configured)?

Reply:
Fair - we can remove this and rely on the `{olm|catalog}.clientCASecret`.

Reply:
This actually isn't even needed if we remove the pprof services.